### ISSUE TITLE
Braintree_php does not work with HHVM because of missing SimpleXMLIterator class

### ISSUE THREAD

**Issue by jaw-sh** [braintree/braintree_php#90]
In my hhvm/error.log file, I am getting these messages when trying to open a page with Braintree PHP calls.

```
\nFatal error: Class undefined: SimpleXMLIterator in /var/www/infinitydev.org/next/vendor/braintree/braintree_php/lib/Braintree/Xml/Parser.php on line 30
```

[Some further investigation](http://stackoverflow.com/questions/21166320/error-unknown-class-simplexmliterator-with-hhvm) would indicate that this is an upcoming feature for HHVM. I don't know if you guys would care to do anything about it right now.

**Comment by sornon1** [comment]
@jaw-sh, which version of the library are you using? Support for HHVM was added in release 3.1 (eliminating the dependency on the simpleXMLIterator). Are you having an issue with version >=3.1?

**Comment by jaw-sh** [comment]
@sornon1 My bad, I should have checked versions before writing this. No idea why I didn't. I just assumed when I googled the problem that it hadn't come out yet, haha.

But yes, actually. I am on version 3.8.1 -- quite a bit higher than 3.1. I've checked HHVM documentation, it should be a part of HHVM's configuration by default.

`var_dump(class_exists("SimpleXMLIterator"));` returns `bool(false)`, so it's definitely not being loaded, which is bizarre. I'm running the default package provided with the Debian 7 apt repositories, so everything should be there.
